float64 x_new
float64 y_new
float64 theta_new
---
float64 x_old
float64 y_old
float64 theta_old